SamplifyAPIClient/SamplifyClient.cs

```diff
@@ -28,7 +30,11 @@ public SamplifyClient(string clientID, string username, string password, Samplif
             {
                 this.APIBaseURL = HostConstants.ProdAPIBaseURL;
                 this.AuthURL = HostConstants.ProdAuthBaseURL;
+            } else if (SamplifyEnv.DEV == env) {
```

Comment:
.NET formatting has different criteria than Java, JS, golang etc. Here we start "else" of "else if" from the next line. If you are using Visual Studio it can take care of formatting automatically, on save. This is a little more important here because it's a public repo.

Reply:
Fixed